tomb.go

```diff
@@ -79,17 +79,17 @@ type Tomb struct {
 
 var (
 	ErrStillAlive = errors.New("tomb: still alive")
-	ErrDying = errors.New("tomb: dying")
+	ErrDying      = errors.New("tomb: dying")
 )
 
 func (t *Tomb) init() {
 	t.m.Lock()
+	defer t.m.Unlock()
```

Comment:
I would only use defer when there are multiple return's possible. It's faster to just call Unlock at the end in this case.